src/utils/ExtensionLoader.js

```diff
@@ -401,24 +401,18 @@ define(function (require, exports, module) {
                     "HtmlEntityCodeHints",
                     "InlineColorEditor",
                     "JavaScriptQuickEdit",
-                    "JSLint",
-                    "LESSSupport",
                     "QuickOpenCSS",
                     "QuickOpenHTML",
                     "QuickOpenJavaScript",
```

Comment:
These `quickopen` modules... They're for quick editing other files from the context of the file that's loaded right? If we're only aiming at thimble, these can probably be removed for now.